# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../twain-dsm/errata.patch
# Copyright (C) 2016 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur twaindsm-2.3.1.orig/TWAIN_DSM/src/twain.h twaindsm-2.3.1.patched/TWAIN_DSM/src/twain.h
--- src/twain.h	2014-05-22 12:35:00.000000000 +0200
+++ src/twain.h	2016-09-06 01:12:30.460170774 +0200
@@ -221,18 +221,10 @@
 /* Numeric types. */
 typedef char           	  TW_INT8,          FAR *pTW_INT8;
 typedef short          	  TW_INT16,         FAR *pTW_INT16;
-#if defined(__APPLE__) /* cf: Mac version of TWAIN.h */
-    typedef int           TW_INT32,         FAR *pTW_INT32;
-#else
-    typedef long          TW_INT32,         FAR *pTW_INT32;
-#endif
+typedef int               TW_INT32,         FAR *pTW_INT32;
 typedef unsigned char     TW_UINT8,         FAR *pTW_UINT8;
 typedef unsigned short    TW_UINT16,        FAR *pTW_UINT16;
-#if defined(__APPLE__) /* cf: Mac version of TWAIN.h */
-    typedef unsigned int  TW_UINT32,        FAR *pTW_UINT32;
-#else
-    typedef unsigned long TW_UINT32,        FAR *pTW_UINT32;
-#endif
+typedef unsigned int      TW_UINT32,        FAR *pTW_UINT32;
 typedef unsigned short    TW_BOOL,          FAR *pTW_BOOL;
 
 
